### PR TITLE
Hide search when draw is active

### PR DESCRIPTION
**Pull request by gjn**
This is an alternative fix for #2511.

### CODE REVIEW

**Comment by oterral** [comment]
easier :)

File: src/style/app.less [hunk 1]
```diff
@@ -205,6 +205,10 @@ button.ga-btn, .ol-control button {
 }
 
 .ga-draw-active {
+
+  #search-container {
+    display: none;
+  }
```
Comment (oterral) [inline src/style/app.less]: Can you add ` #search-container` with other components list which are already hidden when draw is  active see [line 225](https://github.com/geoadmin/mf-geoadmin3/blob/gjn_search_draw/src/style/app.less#L225)

**Comment by gjn** [comment]
Good idea. Done.

**Comment by oterral** [comment]
perfect

**Pull request merged**